sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ReviewSqlQueriesForSecurityVulnerabilities.cs

```diff
@@ -38,29 +38,16 @@
 namespace SonarAnalyzer.Rules.CSharp
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    [Rule(DiagnosticId)]
     public sealed class ReviewSqlQueriesForSecurityVulnerabilities : SonarDiagnosticAnalyzer
     {
-        internal const string DiagnosticId = "S3649";
-        private const string MessageFormat = "Make sure to sanitize the parameters of this SQL command.";
+        internal const string DiagnosticId = "S9999-ucfg-generator";
+        private const string Title = "UCFG generator.";
 
         private static readonly DiagnosticDescriptor rule =
-            DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(rule);
+            DiagnosticDescriptorBuilder.GetUtilityDescriptor(DiagnosticId, Title,
+                new[] { DiagnosticDescriptorBuilder.MainSourceScopeTag });
```

Comment:
- The rule is now "utility", which means that it cannot be configured by the user and always runs.
- The rule should run only on "main" sources.

Reply:
You are now only providing the `MainSourceScopeTag` and not the not configurable one.